token-lending/program/src/state/lending_market.rs

```diff
@@ -65,12 +65,26 @@ const LENDING_MARKET_LEN: usize = 226; // 1 + 1 + 32 + 32 + 32 + 128
 impl Pack for LendingMarket {
```

Comment:
This commit fixes a clippy error for Pack (`pack_into_slice` is supposed to come before `unpack_from_slice`)